### PR TITLE
internalmidi: fix SettingsMainView

### PR DESCRIPTION
**Pull request by hfsfox**

### CODE REVIEW

**Comment by korli** [comment]
Patches should go at https://github.com/HaikuArchives/InternalMIDI/

**Comment by Begasus** [comment]
Patch has been sent upstream in: https://github.com/HaikuArchives/InternalMIDI/pull/3
Also included was the localization there.
Maybe someone should look into finishing that so this could be updated with upstream changes?

**Comment by Begasus** [comment]
@hfsfox could you update the srcGitRev tag and remove the patchset (patch got merged upstream, thanks there @humdingerb ! )

File: haiku-apps/internalmidi/internalmidi-2.5.3.recipe [hunk 1]
```diff
@@ -10,14 +10,15 @@ HOMEPAGE="https://github.com/HaikuArchives/InternalMIDI"
...
+REVISION="2"
 srcGitRev="cb9d90ffb17f0d9d6d1af6b1d6889f4f81d212e3"
 SOURCE_URI="https://github.com/HaikuArchives/InternalMIDI/archive/$srcGitRev.tar.gz"
 CHECKSUM_SHA256="47c7c58f4fcb287f59fa4d106f640aadc9826e51126b0b5c556759c591c5d41d"
 SOURCE_FILENAME="InternalMIDI-$portVersion-$srcGitRev.tar.gz"
 SOURCE_DIR="InternalMIDI-$srcGitRev"
+PATCHES="internalmidi-$portVersion.patchset"
 
-ARCHITECTURES="!all x86_gcc2"
+ARCHITECTURES="!all x86_64 x86_gcc2"
```
[the remark this answers is not in the document]
Comment (Begasus) [reply haiku-apps/internalmidi/internalmidi-2.5.3.recipe]: Done. :)

**Pull request merged**